README.md

```diff
@@ -5,62 +5,78 @@
-- [CNI plugins](./cni-plugin) for Kubernetes to provide highly efficient pod networking and IP Address Management (IPAM).
-- A [Neutron ML2](./networking-calico) plugin to provide VM networking for OpenStack.
-- A [BGP routing stack](https://projectcalico.docs.tigera.io/networking/bgp) that can advertise routes for workload and service IP addresses to physical network infrastructure.
-
-## Getting Started Running Calico
-
-There are many avenues to get started with Calico depending on your situation.
-
-- Trying out Kubernetes on a single host or on your own hardware? The
-  [quick start guide](https://projectcalico.docs.tigera.io/getting-started/kubernetes/quickstart) will have you up and running in
-  about fifteen minutes.
-- Running a managed public cloud? Use our
-  [guides for enabling Calico network policies](https://projectcalico.docs.tigera.io/getting-started/kubernetes/managed-public-cloud/).
-- Want to go deeper? Visit [https://projectcalico.docs.tigera.io](https://projectcalico.docs.tigera.io) for full documentation.
-
-## Getting Started Developing Calico
-
-Calico is an open source project, and welcomes your contribution, be it through code, a bug report, a feature request, or user
-feedback.
-
-- [The Contribution Guidelines](CONTRIBUTING.md) document will get you started on submitting changes to the project.
-- [The Developer Guide](DEVELOPER_GUIDE.md) will walk you through how to set up a development environment, build the code, and run tests.
-- [The Calico Documentation Guide](CONTRIBUTING_DOCS.md) will get you started on making changes to [https://projectcalico.docs.tigera.io](https://projectcalico.docs.tigera.io).
-
-## Join the Calico Community!
-
-The Calico community is committed to fostering an open and welcoming environment, with several ways to engage with other users
-and developers. You can find out more about our monthly meetings, Slack group, and Discourse by visiting our
-[community repository](https://github.com/projectcalico/community).
-
-## License
+<div align=center>
+<img src="https://www.tigera.io/app/uploads/2024/02/Ecosystem_shrunken_2023.svg">
+</div>
 
-Calico is open source, with most code and documentation available under the Apache 2.0 license (see the [LICENSE](/LICENSE.md)), though some elements are necessarily licensed under different open source licenses for compatibility with upstream licensing or code linking. For example, some Calico BPF programs are licensed under GPL v2.0 for compatibility with Linux kernel helper functions.
+## 🤝 Join the Calico Community
+
+- [Calico Big Cats][big-cats]: Become an ambassador and share your journey
+- [Community Meetings][community-meetings]: Engage and contribute
+- [Contribute on GitHub][first-issues]: Start with 'good first issues'
+- [Connect on Slack][slack]: Join the conversation with fellow contributors and our developers
+
+## 💡 Contributing to Project Calico
+
+- [Get Started with Project Calico][get-started]
```

Comment:
It would be great if we could jam in the tutorials page in here too.
```
- [Learn more about Calico][tutorials]

[tutorials](https://www.tigera.io/tutorials/)
```